### DISCUSSION TITLE
Banned User Cannot Be Added Back

### DISCUSSION THREAD

**Discussion by sitehatchery** [owncast/owncast#1499] [Q&A] [answered]
After I have banned a user, the user does not appear in the Banned Users list, so I can't add them back.

![image](https://user-images.githubusercontent.com/2289893/139308993-2571e8ff-3c58-4e45-b249-59a2b046faf1.png)

I banned my own user to test, and now I can't see chat anymore and I can't get it back.

It shows in Firefox, but not Chrome, because it shows as a different user in Firefox. I've cleared cache/cookies to no effect.

**Comment by gabek** [comment, marked as answer]
Filed in https://github.com/owncast/owncast/issues/1518.
